js/Events.js

```diff
@@ -100,8 +100,9 @@ const Events = (() => {
      */
      function on(event, elementId, functionCallback, ...args) {
         //if element provided is string, get the actual element
+        if(!elementId)return;
```

Comment:
Let getElement take care of this imo, it would create less clutter when calling getElement